pkg/webui/console/store/actions/lib.js

```diff
@@ -0,0 +1,40 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/* eslint-disable import/prefer-default-export */
+
+import { createAction } from 'redux-actions'
+
+const createRequestActions = function (
+  baseType,
+  requestPayloadCreator,
+  successPayloadCreator,
+  failurePayloadCreator,
```

Comment:
you dont really need L22-L23 with this implementation. you never set them, so it will always be the identity function.